src/main/java/uk/gov/pay/api/resources/SearchDisputesResource.java

```diff
@@ -34,13 +44,42 @@ public SearchDisputesResource(SearchDisputesService searchDisputesService) {
     @Timed
     @Path("/v1/disputes")
     @Produces(APPLICATION_JSON)
-    public DisputesSearchResults searchDisputes(@Auth Account account,
+    @Operation(security = {@SecurityRequirement(name = "BearerAuth")},
+            operationId = "Search disputes",
+            summary = "Search disputes",
+            description = "Search disputes by status, 'from' and 'to' date, and 'from' and 'to' settled_date. " +
+                    "The Authorisation token needs to be specified in the 'authorization' header " +
+                    "as 'authorization: Bearer YOUR_API_KEY_HERE'",
+            responses = {
+                    @ApiResponse(responseCode = "200", description = "OK",
+                            content = @Content(schema = @Schema(implementation = DisputesSearchResults.class))),
+                    @ApiResponse(responseCode = "401",
+                            description = "Credentials are required to access this resource"),
+                    @ApiResponse(responseCode = "422",
+                            description = "Invalid parameters: from_date, to_date, from_settled_date, to_settled_date, status, display_size. See Public API documentation for the correct data formats",
+                            content = @Content(schema = @Schema(implementation = RequestError.class))),
+                    @ApiResponse(responseCode = "429", description = "Too many requests",
+                            content = @Content(schema = @Schema(implementation = ApiErrorResponse.class))),
+                    @ApiResponse(responseCode = "500", description = "Downstream system error",
+                            content = @Content(schema = @Schema(implementation = RequestError.class)))
+            }
+    )
+    public DisputesSearchResults searchDisputes(@Parameter(hidden = true)
+                                                    @Auth Account account,
+                                                @Parameter(description = "Returns disputes raised on or after the 'from_date' Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - 'YYYY-MM-DDTHH:MM:SSZ'.")
                                                 @QueryParam("from_date") String fromDate,
+                                                @Parameter(description = "Returns disputes raised before the 'to_date' Date and time must be coordinated Universal Time (UTC) and ISO 8601 format to second-level accuracy - 'YYYY-MM-DDTHH:MM:SSZ'.")
                                                 @QueryParam("to_date") String toDate,
+                                                @Parameter(description = "Returns disputes settled on or after the 'from_settled_date'. Date must be in ISO 8601 format to date-level accuracy - 'YYYY-MM-DD'. Disputes are settled when your payment service provider takes the disputed amount from a payout to your bank account.<br><br>You can only use 'from_settled_date' to find payments raised on or after 8 August 2022.")
```

Comment:
I think this should be "disputes raised" rather than "payment raised". What you have here is the same as the tech docs PR so I'll add a comment on that too.

Reply:
👍